doc/whats_new.rst

```diff
@@ -99,6 +99,8 @@ Bug
 
 - Fix bug with channel names in ``mgh70`` montage in :func:`mne.channels.read_montage` by `Eric Larson`_
 
+- Fix bug in :func:`mne.io.read_raw_edf` where GDF files had ``info['highpass']`` and ``info['lowpass']`` set to NaN by `Eric Larson`_
```

Comment:
do you want to add that `meas_date` has also changed?

Reply:
ahh yes I should add it